_posts/en/newsletters/2021-02-24-newsletter.md

```diff
@@ -0,0 +1,110 @@
+---
+title: 'Bitcoin Optech Newsletter #137'
+permalink: /en/newsletters/2021/02/24/
+name: 2021-02-24-newsletter
+slug: 2021-02-24-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter describes the results of discussion about
+choosing activation parameters for a taproot soft fork and includes our
+regular sections with selected questions and answers from the Bitcoin
+StackExchange, releases and release candidates, and notable changes to
+popular Bitcoin infrastructure software.
+
+## News
+
+- **Taproot activation discussion:** Michael Folkson [summarized][folkson
+  lot] a second meeting about activation parameters for [taproot][topic
+  taproot], concluding that the meeting was "largely unproductive" about
+  choosing what value to use for the LockinOnTimeout (LOT) parameter of
+  [BIP8][].  However, there was near universal agreement for other
+  activation parameters, most notably reducing the amount of hash rate
+  that needs to signal for the fork to activate from 95% to 90%.
+
+    Discussion continued about the LOT parameter on the mailing list,
+    mainly about whether or not it should be exposed to users as an
+    option they can set (or are forced to set).  No clear agreement was
+    reached as of this writing and there does not seem to be a widely
```

Comment:
Repeat of "widely" here and at end of sentence. There isn't overwhelming consensus (that's for sure) but there were more Core contributor effective NACKs for lot=true than false. I would say the open questions at this point are whether Core releases anything (ie leaves it to "community releases") and if Core does release lot=false whether there is an alternative "community release" that sets lot=true.